src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -362,6 +402,10 @@ public static function getFeatured(array $owner, int $page = null)
 
 		$data['orderedItems'] = $list;
 
+		if (!empty($cachekey)) {
```

Comment:
According to the description of the parameter, I believe you should have checked the value of `$nocache` here.